fews_3di/simulation.py

```diff
@@ -149,25 +149,41 @@ def run(self):
         self.simulation_id, self.simulation_url = self._create_simulation(model_id)
 
         laterals_csv = self.settings.base_dir / "input" / "lateral.csv"
-        laterals = utils.lateral_timeseries(laterals_csv, self.settings)
-        self._add_laterals(laterals)
+        if path.exists(laterals_csv):
+            laterals = utils.lateral_timeseries(laterals_csv, self.settings)
+            self._add_laterals(laterals)
+        else:
+            logger.info("No lateral timeseries found, skipping...")
 
         saved_state_id_file = self.settings.base_dir / SAVED_STATE_ID_FILENAME
         if self.settings.save_state:
             self._add_initial_state(saved_state_id_file)
             self.saved_state_id = self._prepare_initial_state()
+        else:
+            logger.info("Not using saved state, skipping...")
 
         rain_file = self.settings.base_dir / "input" / "precipitation.nc"
-        rain_raster_netcdf = utils.write_netcdf_with_time_indexes(
-            rain_file, self.settings
-        )
-        self._add_rain(rain_raster_netcdf)
+        if path.exists(rain_file):
+            rain_raster_netcdf = utils.write_netcdf_with_time_indexes(
+                rain_file, self.settings
+            )
+            self._add_rain(rain_raster_netcdf)
+        else:
+            logger.info("No rain file found, skipping...")
```

Comment:
Die `...` gebruik ik als er daarna iets gebeurd dat lang duurt. "Berekening wordt nu gestart...". Als het een normale mededeling is gewoon een punt gebruiken.

Consistentie is belangrijk :-)